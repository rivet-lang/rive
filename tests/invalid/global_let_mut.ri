let mut MUTABLE_STATIC: i32 = 3;

fn main() {
    let x = MUTABLE_STATIC;
}
