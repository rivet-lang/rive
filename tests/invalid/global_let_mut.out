tests/invalid/global_let_mut.ri:4:13: error: use of mutable global objects is unsafe and requires `unsafe` block
   = note: mutable global objects can be mutated by multiple threads: aliasing violations
   or data races will cause undefined behavior
rivetc: error: could not compile package `global_let_mut`, aborting due to previous error